\hypertarget{classdjango_1_1contrib_1_1sessions_1_1backends_1_1base_1_1_create_error}{}\doxysection{django.\+contrib.\+sessions.\+backends.\+base.\+Create\+Error Class Reference}
\label{classdjango_1_1contrib_1_1sessions_1_1backends_1_1base_1_1_create_error}\index{django.contrib.sessions.backends.base.CreateError@{django.contrib.sessions.backends.base.CreateError}}


Inheritance diagram for django.\+contrib.\+sessions.\+backends.\+base.\+Create\+Error\+:
% FIG 0


Collaboration diagram for django.\+contrib.\+sessions.\+backends.\+base.\+Create\+Error\+:
% FIG 1


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Used internally as a consistent exception type to catch from save (see the
docstring for SessionBase.save() for details).
\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/\+Lib/site-\/packages/django/contrib/sessions/backends/base.\+py\end{DoxyCompactItemize}
